app/pages/reset-password/submit-email-form.spec.js

```diff
@@ -42,16 +42,14 @@ describe('SubmitEmailForm', function () {
   it('conditionally shows and hides an email success icon and message', function () {
     wrapper.setProps({ emailSuccess: true });
     assert.equal(wrapper.find('i.form-help.success').length, 1);
-    assert.equal(wrapper.find('Translate').last().prop('content'), 'resetPassword.emailSuccess');
+    assert.equal(wrapper.find('#reset-status').find('Translate').prop('content'), 'resetPassword.emailSuccess');
     wrapper.setProps({ emailSuccess: false });
     assert.equal(wrapper.find('i.form-help.success').length, 0);
-    assert.equal(wrapper.find('Translate').length, 1);
   });
 
   it('conditionally shows and hides an email error message', function () {
     wrapper.setProps({ emailError: 'test error message' });
-    assert.equal(wrapper.find('Translate').last().prop('content'), 'resetPassword.emailError');
+    assert.equal(wrapper.find('#reset-status').find('Translate').last().prop('content'), 'resetPassword.emailError');
```

Comment:
You can use attribute selectors to find a specific Translate component here too.